packages/gatsby-theme-styleguide/src/templates/preview-page.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { graphql } from 'gatsby'
+
+export default ({ data }) => {
+  return (
+    <>
```

Comment:
We should use `Helmet` to add styles in the `head`, and is possible inject the html inside the `body` tag

Reply:
Helmet added